src/lizaalert/courses/views.py

```diff
@@ -44,6 +44,10 @@ def get_queryset(self):
         users_annotations - provide annotations only for auth user.
         """
         user = self.request.user
+        course_id = self.kwargs.get("pk")
+        course = None
+        if course_id:
+            course = get_object_or_404(Course, id=course_id)
```

Comment:
если курс не найден, то 404 это ок

```suggestion
        course = get_object_or_404(Course, id= self.kwargs.get("pk", None))
```

Reply:
Такое изменение даст ошибку 404 по ручке /courses/